modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemObjectProvider.java

```diff
@@ -131,52 +130,30 @@ private FileEntry _getInfoItem(
 				"Unsupported info item identifier " + infoItemIdentifier);
 		}
 
-		long classPK = 0;
+		try {
+			if (infoItemIdentifier instanceof ClassPKInfoItemIdentifier) {
+				ClassPKInfoItemIdentifier classPKInfoItemIdentifier =
+					(ClassPKInfoItemIdentifier)infoItemIdentifier;
 
-		if (infoItemIdentifier instanceof ClassPKInfoItemIdentifier) {
-			ClassPKInfoItemIdentifier classPKInfoItemIdentifier =
-				(ClassPKInfoItemIdentifier)infoItemIdentifier;
+				return _dlAppLocalService.fetchFileEntry(
```

Comment:
As we are using a fetch method in this class, shouldn't be throw an exception in case of null instead o return null as we are doing in other cases?

Reply:
I'd prefer to approve this.

We are doing that in our current master, in fact